ci/kind/kind-setup.sh

```diff
@@ -288,7 +289,32 @@ function configure_vlan_subnets {
 
     docker_run_with_host_net iptables -t filter -A FORWARD -i $bridge_interface -o $vlan_interface -j ACCEPT
     docker_run_with_host_net iptables -t filter -A FORWARD -o $bridge_interface -i $vlan_interface -j ACCEPT
+    ipam_vlan_interface+=($vlan_interface)
   done
+
+  if [[ $FLEXIBLE_IPAM == true ]]; then
+    docker_run_with_host_net ipset create excluded_subnets hash:net
+    docker_run_with_host_net ipset add excluded_subnets 192.168.241.0/24
+    docker_run_with_host_net ipset add excluded_subnets 192.168.242.0/24
+    docker_run_with_host_net ipset add excluded_subnets 192.168.240.0/24
+    docker_run_with_host_net ipset list excluded_subnets
+    
+    # Bypass default Docker SNAT rule for FlexibleIPAM traffic from the untagged subnet (192.168.240.0/24, which is the subnet for the Docker bridge network)
+    # and destined to the VLAN subnets (192.168.241.0/24, 192.168.242.0/24).
+    docker_run_with_host_net iptables -t nat -I POSTROUTING 1 ! -o $bridge_interface -s 192.168.240.0/24 -m set --match-set excluded_subnets dst -j RETURN
+
+    # With FlexibleIPAM, Antrea SNAT is disabled (noSNAT: true) so Pods don't have access to the external network by default (including regular / NodeIPAM Pods).
+    # Our e2e tests require external network access for regular Pods, so we need to add a custom SNAT rule.
+    docker_run_with_host_net iptables -t nat -A POSTROUTING ! -o $bridge_interface -s 10.244.0.0/16 -m set ! --match-set excluded_subnets dst -j MASQUERADE
+
+    # Allow traffic between the two VLAN subnets
+    docker_run_with_host_net iptables -I FORWARD -i ${ipam_vlan_interface[0]} -o ${ipam_vlan_interface[1]} -j ACCEPT 
+    docker_run_with_host_net iptables -I FORWARD -i ${ipam_vlan_interface[1]} -o ${ipam_vlan_interface[0]} -j ACCEPT 
```

Comment:
it would probably be better not to add yet another assumption that `len(ipam_vlan_interface) == 2`.

couldn't we have something like this:
```
docker_run_with_host_net iptables -I FORWARD -i ${iface_name}.+ -o ${iface_name}.+ -j ACCEPT 
```

`iptables` supports prefix matching with `+`. i assume that also works for VLAN sub-interfaces.